manage.py

```diff
@@ -6,13 +6,37 @@
 #!/usr/bin/env python
 import os
 import sys
+import uuid
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "infohub.settings")
 
     from django.core.management import execute_from_command_line
 
+     # hack to prevent admin prompt
+    if len(sys.argv) == 2 and sys.argv[1] == 'syncdb':
+        sys.argv.append('--noinput')
+
     execute_from_command_line(sys.argv)
-    
-    
+
+    #additional code to create superuser 
+
+    if 'syncdb' in sys.argv:
+    	from django.contrib.auth.models import User
+        admin_id = 'admin'
+        admin_email = "admin@superuser.com"
+        admin_password = "admin_pass"
+
+
+		 # admin exists?
+        user_list = User.objects.filter(username=admin_id)
+        if len(user_list) == 0: 
+            print 'create superuser: ' + admin_id
+            new_admin = User.objects.create_superuser(admin_id, admin_email,admin_password)
```

Comment:
do we want to hardcode admin_id, email, password? 
Would it not be better to have a config file from where these id, email, pwds are picked up from ? that ways it will remain configurable